swarm/storage/mru/resource_tmp_test.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2018 The go-ethereum Authors
```

Comment:
why is this tmp again?

Reply:
It was because of potential future conflicts when I wrote it. Of which there are plenty anyway. I'm currently underway rebasing the pss notify + resource one, so can we please PR merging this code to `resource_test.go` after that? Thx